src/main/java/com/vackosar/gitflowincrementalbuild/control/DifferentFiles.java

```diff
@@ -62,12 +80,25 @@
                 paths.addAll(worker.getChangesFromStatus());
             }
         } finally {
+            if (worker != null) {
+                worker.credentialsProvider.resetAll();
+            }
             git.getRepository().close();
             git.close();
         }
         return paths;
     }
 
+    /**
+     * Only for testing!
+     *
+     * @param key environment key for {@link DelegatingCredentialsProvider#lookupCredentials(URIish)}
+     * @param value environment value for {@link DelegatingCredentialsProvider#lookupCredentials(URIish)}
+     */
+    void putAdditionalNativeGitEnvironment(String key, String value) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/gitflow-incremental-builder/gitflow-incremental-builder/pullRequest?prid=4845076)